packages/solo/src/pipe-entrypoint.js

```diff
@@ -1,7 +1,7 @@
 /* global process */
 // @ts-check
 import '@endo/init/pre-bundle-source.js';
-import '@endo/init';
+import '@endo/init/legacy.js';
```

Comment:
`pipe-entrypoint.js` is just a wrapper around cosmic-swingset's sim-chain, so I'm surprised that this isn't just:

```js
import '@endo/init/unsafe-fast.js';
```

If you've tested with something like this and found it didn't work, then feel free to disregard, otherwise please do give it a shot.

Reply:
> I'm surprised that this isn't just:
> 
> ```js
> import '@endo/init/unsafe-fast.js';
> ```
> 
> If you've tested with something like this and found it didn't work

I have not tested, but it wouldn't work because unsafe-fast doesn't set the needed overrideTaming: severe. In theory we should have both, but that isn't an entry point that exists

Reply:
Should be one or the other, but not both. unsafe-fast suppressed the freezing that harden does. Without that freezing, override tamings are irrelevant. Perhaps unsafe-fast should even do `overrideTaming: 'min'` ?

Reply:
Oh right, silly me! I'll give it a shot.

Reply:
Verified `unsafe-fast` works, as it should.